commandlog.txt

```diff
@@ -0,0 +1,40 @@
+    1  dir
```

Comment:
can you remove this file from the github and keep some condensed commands in your notes?